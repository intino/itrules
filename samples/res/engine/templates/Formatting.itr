defrule type(Person)
    Default: $Name
    UpperCase: $Name+Uppercase
    LowerCase: $Name+Lowercase
    CamelCase: $Name+Camelcase
    LowerCamelCase: $Name+LowerCamelcase

    Year: $Birthday+Year
    MonthYear: $Birthday+MonthYear
    ShortDate: $Birthday+ShortDate
    FullDate: $Birthday+FullDate
    DayOfWeek: $Birthday+DayOfWeek
    Time: $Birthday+Time
endrule