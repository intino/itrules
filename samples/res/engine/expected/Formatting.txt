Default: Pau Gasol
UpperCase: PAU GASOL
LowerCase: pau gasol
CamelCase: PauGasol
LowerCamelCase: pauGasol

Default: Sun Jul 06 00:00:00 WEST 1980
Year: 1980
MonthYear: July 1980
ShortDate: 06/07/1980
FullDate: 06 July 1980
DayOfWeek: Sunday
Time: 00:00
